build.gradle

```diff
@@ -1,106 +1,185 @@
-buildscript {
-    ext {
-        grailsVersion = project.grailsVersion
-    }
-    repositories {
-        mavenLocal()
-        maven { url "https://repo.grails.org/grails/core" }
-    }
-    dependencies {
-        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-    }
+plugins {
+    id "groovy"
+    id "org.grails.grails-plugin"
+    id "org.grails.grails-gsp"
+    id "org.grails.grails-web"
+    id "idea"
+    id "com.gorylenko.gradle-git-properties"
+    id "application"
+    id "eclipse"
+    id "java-library"
+    id "maven-publish"
 }
 
-version "2.0.0-RC1"
 group "org.grails.plugins"
 
-apply plugin:"eclipse"
-apply plugin:"idea"
-apply plugin:"org.grails.grails-plugin"
-apply plugin:"org.grails.grails-plugin-publish"
-
-ext {
-    grailsVersion = project.grailsVersion
-    gradleWrapperVersion = project.gradleWrapperVersion
-}
-
 repositories {
     mavenLocal()
+    mavenCentral()
     maven { url "https://repo.grails.org/grails/repo" }
+    maven { url "https://repo.grails.org/grails/core" }
+    maven { url "https://build.shibboleth.net/nexus/content/repositories/releases" }
+    maven { url "https://build.shibboleth.net/nexus/content/groups/public/" }
+    maven { url "https://code.lds.org/nexus/content/groups/main-repo" }
+    maven { url "https://repository.jboss.org/maven2/" }
+    maven { url "http://jasperreports.sourceforge.net/maven2"
+        allowInsecureProtocol = true
+    }
+    maven { url "http://10.100.3.6:8080/nexus/content/repositories/common-component-snapshots"
```

Comment:
Is this a private URL? If yes then please remove it

Reply:
Yes will remove the same.